specs/3PhaseACMeasurement/doc/spec.md

```diff
@@ -0,0 +1,951 @@
+such as the different power types (active, reactive and apparent power) there is
+an attribute for the total from all phases e.g. `totalActivePower` and an
+attribute for values from each phase as a structured value e.g. `activePower`.
+There are also attributes for different energy types: active (delivered net
+energy), reactive and apparent. They are cumulative values and the start time
+for their measurement can be saved to the `dateEnergyMeteringStarted` attribute.
+
+For most of the attributes there are various ways they can be actually measured.
+For this purpose the `measurementType` metadata attribute can be used. It can
+have the following values:
+
+-   instant: The value is from the specific instant of time
+-   average: The value is the average of a time period
+-   rms: The value is the root mean square of a time period
+-   maximum: The value is the maximum of a time period
+-   minimum: The value is the minimum of a time period
+
+When using the average, rms, mininum or maximum values another metadata
+attribute called `measurementInterval` should be used to give the length of the
+measurement period in seconds. Also the `timestamp` metadata attribute should be
+the end time of the measurement period.
+
+## Data Model
+
+A JSON Schema corresponding to this data model can be found
+[here](../schema.json).
+
+-   `id` : Entity's unique identifier.
+
+-   `type` : It must be equal to `3PhaseACMeasurement`.
+
+-   `source` : A sequence of characters giving the source of the entity data.
+
+    -   Attribute type: Text or URL
+    -   Optional
+
+-   `dataProvider` : Specifies the URL to information about the provider of this
+    information
+
+    -   Attribute type: URL
+    -   Optional
+
+-   `name` : Name given to the measurement location or target.
+
+    -   Normative References: [https://schema.org/name](https://schema.org/name)
+    -   Optional
+
+-   `description` : A longer description of the measurement.
+
+    -   Normative References:
+        [https://schema.org/description](https://schema.org/description)
+    -   Optional
+
+-   `location` : location of the measurement target represented by a GeoJSON
+    point.
+
+    -   Attribute type: `geo:json`.
+    -   Normative References:
+        [https://tools.ietf.org/html/draft-ietf-geojson-03](https://tools.ietf.org/html/draft-ietf-geojson-03)
+    -   Optional
+
+-   `address` : Civic address where the measurement target is located.
+
+    -   Normative References:
+        [https://schema.org/address](https://schema.org/address)
+    -   Optional
+
+-   `areaServed` : Higher level area to which the measurement belongs to. It can
+    be used to group per responsible, district, neighbourhood, etc.
+
+    -   Normative References:
+        [https://schema.org/areaServed](https://schema.org/areaServed)
+    -   Optional
+
+-   `refDevice` : Reference to the device(s) measured or the devices that made
```

Comment:
mmm I think it would be better to differentiate between both i.e the device measured and the device used to get the measurement. As per other data models I would use

`refDevice` : Device used to obtain the measurement

`refTargetDevice` : Device for which the measurement was taken. I would, by the way, check how SAREF names this property, and if there is a coincidence in semantics, I would suggest to align as much as possible with SAREF.

Reply:
For now I added the refTargetDevice attribute. From SAREF I found the [is measured by device](http://ontology.tno.nl/saref/saref_isMeasuredByDevice.html)
property which could replace refDevice very nicely. There was also the [is controlled by device](http://ontology.tno.nl/saref/saref_isControlledByDevice.html)
property which could be used to replace refTargetDevice but might not be very suitable.